extensions/amp-analytics/0.1/vendors/deepbi.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {jsonLiteral} from '../../../../src/json';
+
+const DEEPBI_CONFIG = jsonLiteral({
+  'vars': {
+    'gdpr': true,
+    'endpoint': 'api.deep.bi',
+  },
+  'requests': {
+    'base':
+      'https://${endpoint}/v1/pixel/${datasetKey}/p.gif?accessKey=${accessKey}&gdpr=${gdpr}&page_href=${sourceUrl}&referrer_href=${documentReferrer}&page_title=${title}&amp=true',
+    'pageview': '${base}&event_type=page-open',
+    'event': '${base}&event_type=event',
+  },
+  'triggers': {
+    'defaultPageview': {
+      'on': 'visible',
+      'request': 'pageview',
+    },
+  },
+  'transport': {
+    'beacon': false,
+    'xhrpost': false,
+    'image': true,
```

Comment:
Just to confirm that the server only handles get request. AMP handles the fallback to `xhrpost` and `image` if the browser doesn't support one transportation method. So we'd recommend using the default transport method if that's supported by server. Thanks.

Reply:
Hello @zhouyx ,
Thank you very much for your input! 
Right now we would like to implement the `image` option as it is well tested and robust here. 
If it is absolutely necessary, we'll implement other transport methods as well, but our initial idea was to start the simplest way and when we are more experienced with AMP go for the full stack.
Please let me know what you think and if we can continue with the current version.
THanks a lot!!!

Reply:
@pkalinowski That sounds good. When you do implement the other transport methods, just make another PR to change this config :)

Reply:
Thank you very much, we'll do!